### PR TITLE
Update Hong Kong Baptist University

### PR DESCRIPTION
**Pull request by xu-cheng**
# Contributing to CSrankings

Thanks for contributing to CSrankings! Here are some guidelines to getting your pull request accepted.

0. Use a reasonable title that explains what the PR corresponds to (as in, not "Update csrankings-1.csv").

1. _Do not use Excel to edit any .csv files; Excel incorrectly tries to
convert some Google Scholar entries to formulas, corrupting the
database. Use a text editor like emacs or NotePad instead._

2. _Insert new faculty **in alphabetical order**, not at the end of `csrankings-[0-9].csv`._ **Do not modify `csrankings.csv`, which is auto-generated.**

3. _Read and check **all** the boxes below by filling them in with an X._

**Inclusion criteria**

- [x] Make sure that any faculty you add meet the inclusion
criteria. Eligible faculty include only full-time, tenure-track research
faculty members on a given campus who can *solely* advise PhD students in
Computer Science. Faculty not in a CS department or similar who can
advise PhD students in CS can be included regardless of their home
department. Faculty should also have a 75%+ time appointment (check
`old/industry.csv` for faculty who are now more than 25% in industry).

**Updating an affiliation or home page**

- [x] Update affiliations, home pages, and Google Scholar entries by modifying `csrankings-[0-9].csv`. For the Google Scholar entry, just use the alphanumeric identifier in the middle of the URL. If none is there, put NOSCHOLARPAGE.

**Adding one or more faculty members (including an entire department)**

- [x] If the department is not yet listed in CSrankings, the entire faculty needs to be added (not just one faculty member).

- [x] Enter each faculty member's [DBLP](http://dblp.org) name, home page, and Google Scholar entry (just the alphanumeric identifier, not the whole URL) by modifying `csrankings-[0-9].csv`; include disambiguation suffixes like 0001 as needed. If the faculty entry is currently ambiguous, please do not include them. Send mail to the DBLP maintainers (dblp@dagstuhl.de) with a few publications by a particular faculty member; also, open an issue so that when the DBLP database is updated, that faculty member's information can be added.

- [x] If DBLP has multiple entries for this person, all of them need to be listed. If an alias is not already present in `dblp-aliases.csv`, add it.

- [x] If the institution you are adding is not in the US,
update `country-info.csv`.

**(Advanced) Quick contribution via a shallow clone** 

A full clone of the CSrankings repository is almost 2GB.  To contribute a change without creating a full local clone of the CSrankings repo, you can do a _shallow clone_.  To do so, follow these steps:

1. Fork the CSrankings repo.  If you have an existing fork, but it is not up to date with the main repository, this technique may not work.  If necessary, delete and re-create your fork to get it up to date.  (Do **not** delete your existing fork if it has unmerged changes you want to preserve!)
2. Do a shallow clone of your fork: `git clone --depth 1 https://github.com/yourusername/CSrankings`.  This will only download the most recent commit, not the full git history.
3. Make your changes on a branch, push them to your clone, and create a pull request on GitHub as usual.

If you want to do another contribution and some time has passed, perform steps 1-3 again, creating a fresh fork and shallow clone.

### CODE REVIEW

File: csrankings-5.csv [hunk 1]
```diff
@@ -2179,8 +2179,8 @@ Michael Jenkin,York University,http://eecs.lassonde.yorku.ca/faculty/michael-r-m
 Michael Johnson,Macquarie University,http://web.science.mq.edu.au/~mike,NOSCHOLARPAGE
 Michael Julian Black [IS],Max Planck Society,https://ps.is.tuebingen.mpg.de/person/black,6NjbexEAAAAJ
 Michael Jurczyk,University of Missouri,https://engineering.missouri.edu/faculty/michael-jurczyk,NOSCHOLARPAGE
-Michael K. Ng,Hong Kong Baptist University,http://www.math.hkbu.edu.hk/~mng,BBpjLiIAAAAJ
-Michael K. P. Ng,Hong Kong Baptist University,http://www.math.hkbu.edu.hk/~mng,BBpjLiIAAAAJ
+Michael K. Ng,University of Hong Kong,https://hkumath.hku.hk/~mng/,BBpjLiIAAAAJ
+Michael K. P. Ng,University of Hong Kong,https://hkumath.hku.hk/~mng/,BBpjLiIAAAAJ
```
Comment (emeryberger) [inline csrankings-5.csv]: Not in CS, not listed here: https://www.cs.hku.hk/people/academic-staff.

File: csrankings-4.csv [hunk 2]
```diff
@@ -338,6 +338,7 @@ Jihad Mohamad Jaam,Qatar University,http://qspace.qu.edu.qa/handle/10576/61/disc
 Jihong Kim 0001,Seoul National University,http://cares.snu.ac.kr,E9A5Rv8AAAAJ
 Jihong Zhu,Tsinghua University,http://www.cs.tsinghua.edu.cn/publish/csen/4923/2014/20140108153546275916040/20140108153546275916040_.html,3K-zt_oAAAAJ
 Jihun Hamm,Tulane University,http://www.cs.tulane.edu/~jhamm3,DdRsMKsAAAAJ
+Jiji Zhang,Hong Kong Baptist University,https://rel.hkbu.edu.hk/people/zhang-jiji,5GAikocAAAAJ
```
Comment (emeryberger) [inline csrankings-4.csv]: Not in CS (Religion & Philosophy).

File: csrankings-5.csv [hunk 3]
```diff
@@ -2197,7 +2197,7 @@ Michael Kirley,University of Melbourne,http://people.eng.unimelb.edu.au/mkirley,
 Michael Koch 0001,Bundeswehr University Munich,http://www.unibw.de/michael.koch,knY18cEAAAAJ
 Michael Kohlhase,University of Erlangen–Nuremberg,https://kwarc.info/people/mkohlhase,H0xjVyoAAAAJ
 Michael Krone,University of Tübingen,https://uni-tuebingen.de/en/faculties/faculty-of-science/departments/computer-science/lehrstuehle/big-data-visual-analytics-in-life-sciences/group-members/jun-prof-dr-michael-krone,srDA4dAAAAAJ
-Michael Kwok-Po Ng,Hong Kong Baptist University,http://www.math.hkbu.edu.hk/~mng,BBpjLiIAAAAJ
+Michael Kwok-Po Ng,University of Hong Kong,https://hkumath.hku.hk/~mng/,BBpjLiIAAAAJ
```
Comment (emeryberger) [inline csrankings-5.csv]: As above, not in CS.

File: csrankings-9.csv [hunk 4]
```diff
@@ -1225,6 +1225,7 @@ Xue Chen 0001,George Mason University,https://sites.google.com/view/xuechencs/ho
 Xue Dong Yang,University of Regina,http://www.cs.uregina.ca/People/Profiles/XueDongYang.html,NOSCHOLARPAGE
 Xue Li 0001,University of Queensland,http://staff.itee.uq.edu.au/xue,3eIIFWIAAAAJ
 Xue Lin,Northeastern University,https://web.northeastern.edu/xuelin,p87KNLIAAAAJ
+Xue-Cheng Tai,Hong Kong Baptist University,http://www.math.hkbu.edu.hk/~xuechengtai/HKBU.html,XOf_txgAAAAJ
```
Comment (emeryberger) [inline csrankings-9.csv]: Math, not CS.

File: csrankings-9.csv [hunk 5]
```diff
@@ -1316,6 +1317,7 @@ Yang Hua,Queen's University Belfast,https://www.qub.ac.uk/schools/eeecs/Connect/
 Yang Liu 0003,Nanyang Technological University,http://www.ntu.edu.sg/home/yangliu,NOSCHOLARPAGE
 Yang Liu 0004,University of Texas at Dallas,http://www.hlt.utdallas.edu/~yangl,NOSCHOLARPAGE
 Yang Liu 0005,Tsinghua University,http://nlp.csai.tsinghua.edu.cn/~ly,NOSCHOLARPAGE
+Yang Liu 0007,Hong Kong Baptist University,http://www.comp.hkbu.edu.hk/~csygliu/,NOSCHOLARPAGE
```
Comment (emeryberger) [inline csrankings-9.csv:1322]: Listed as "research" professor, but only tenure-track faculty are currently eligible for inclusion.

**Review by emeryberger — CHANGES_REQUESTED** [review]

File: csrankings-5.csv (hunk 1 above)
Comment (xu-cheng) [reply csrankings-5.csv]: This is to update affiliation of existing entry. I can remove it if you want.

File: csrankings-4.csv (hunk 2 above)
Comment (xu-cheng) [reply csrankings-4.csv]: While being another department, his recent research area is heavily focused on machine learning and AI. You can see  https://scholar.google.com/citations?user=5GAikocAAAAJ&hl=en for more info.

File: csrankings-9.csv (hunk 4 above)
Comment (xu-cheng) [reply csrankings-9.csv]: While being another department, his research area is heavily focused on image processing and computer vision.

File: csrankings-9.csv (hunk 5 above)
Comment (xu-cheng) [reply csrankings-9.csv:1322]: The personal webpage is outdated. He is now an assistant professor. See https://www.comp.hkbu.edu.hk/v1/?page=faculty

File: csrankings-9.csv (hunk 4 above)
Comment (xu-cheng) [reply csrankings-9.csv]: I have removed this entry.

File: csrankings-4.csv (hunk 2 above)
Comment (xu-cheng) [reply csrankings-4.csv]: I have removed this entry.

File: csrankings-9.csv (hunk 5 above)
Comment (xu-cheng) [reply csrankings-9.csv:1322]: I have removed this entry.

**Comment by xu-cheng** [comment]
@emeryberger Hi, any chance to merge this PR? Thanks

**Comment by emeryberger** [comment]
The files `csrankings-[0-9].csv` are now obsolete. Please use `csrankings-[a-z].csv` and try again. Sorry for the inconvenience. Thanks.

**Pull request closed without merging**